Set-Video-Mode/README.md

```diff
@@ -0,0 +1,105 @@
+# script-plugin-set-video-mode
+
+This repository contains a example of using Script Plugin and Plugin Message feature to set BrightSign VideoMode.
+
+videomode_plugin.brs is a custom BrightSign plugin example that you can configure videoMode for series 4 players via Plugin Message Usage.
+
+## Plugin Message Usage
+
+1. Update videomode_plugin.brs with your expected screens settings
+2. Add videomode_plugin.brs to BrightAuthor or BrightAuthor:Connected (Presentation Settings > Support Content > Script Plugin)
+3. Name ```videomode``` as your Script plugin name (which must be the same as the <plugin_name> described in your brs plugin initialization function)
```

Comment:
This is slightly confusing to read. Could this be reworded with direct references to what should be assigned or named?

Reply:
This line 11 was added based on docs "... Click the Add Script Plugin button, give the plugin a name (which must be the same as the <plugin_name> described in the initialization function below..." from [BrightAuthor Plugins and Parsers](https://brightsign.atlassian.net/wiki/spaces/DOC/pages/370673619/BrightAuthor+Plugins+and+Parsers). 

Does "3. Assign the name 'videomode' to your Script plugin. Make sure that the assigned name matches the <plugin_name> specified in your BRs plugin initialization function." this sound reasonable to you?

Reply:
Updated in [162fa7e](https://github.com/brightsign/BrightAuthor-Plugins/pull/20/commits/162fa7ebf3ab7936aa73d8d76479607b8c60f497)

Reply:
@mxin512 Just curious are there limitations to see the video tab? I don't see it on my player's DWS.

Reply:
@mingjun7 I believe that if you are using an audio only player, you won't see the Video Tab. Otherwise, you should see the Video Tab for DWS.